From 3b41c1a26d4d1e93485a0871a089bb9f3fcd2205 Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.1-3b41c1a26d4-20220304T093323Z-avarab@gmail.com>
In-Reply-To: <20220304085719.yht6hiuybhgfd6vo@fs>
References: <20220304085719.yht6hiuybhgfd6vo@fs>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 4 Mar 2022 10:29:31 +0100
Subject: [PATCH] log tests: fix "abort tests early" regression in ff37a60c369
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix a regression in ff37a60c369 (log tests: check if grep_config() is
called by "log"-like cmds, 2022-02-16), a "test_done" command used
during development made it into a submitted patch causing tests 41-136
in t/t4202-log.sh to be skipped.

Reported-by: Fabian Stelzer <fs@gigacodes.de>
Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 t/t4202-log.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/t/t4202-log.sh b/t/t4202-log.sh
index 55fac644464..46e413bcc93 100755
--- a/t/t4202-log.sh
+++ b/t/t4202-log.sh
@@ -484,7 +484,6 @@ do
 		)
 	'
 done
-test_done
 
 test_expect_success 'log --author' '
 	cat >expect <<-\EOF &&
-- 
2.35.1.1230.ga6e6579e98c

